rfcs/63-nursery.md

```diff
@@ -0,0 +1,168 @@
+# Feature Name: `bevy-nursery`
+
+## Summary
+
+Work done within the [broader Bevy ecosystem](https://bevyengine.org/assets/) is sometimes a good fit as part of the core engine.
+These crates should live in a **nursery** owned by the Bevy org for a period of time,
+where they can be polished and refined by the broader community,
+before eventually being merged in completely.
+
+## Motivation
+
+Third-party crates, commonly plugins, are often used to extend the engine in ways that are broadly applicable.
+Polished crates that are a good fit for the engine should eventually be merged in wholesale,
+rather than reimplemented, to avoid the [second system effect](https://en.wikipedia.org/wiki/Second-system_effect)
+and minimize breakage for their users.
+
+However, merging in these crates directly will result in very large, unreviewable PRs,
+and may compromise Bevy standards for quality.
+But if we delay, critical ecosystem crates may go stale, pointlessly splitting efforts, or making it hard for beginners to discover the de-facto standards.
+
+## User-facing explanation
+
+In addition to the core crates included within `bevy` itself, there are a number of **nursery** crates,
+which are being prepared for inclusion in the engine.
+
+These crates follow exactly the same technical rules as any other third-party crate, but:
+
+1. Live in the [`bevyengine` organization](https://github.com/bevyengine) on GitHub.
+2. Follow Bevy's standards for code quality and project management.
+3. Explicitly welcome community contributions to help get them ready for release.
+4. Will have a branch prepared for the newest release of Bevy before we release a new major version.
```

Comment:
I have a few questions about this:

1- If some `bevy` repo PR introduce a breaking change into some `nursery` crate, there will besome defined process to detect and warn `nursery` crate maintainers? Like a custom CI on every `nursery` crate which will detect it?

2- What happens if at the end of the 3-month cycle, `bevy` is ready for a new release, but some `nursery` crates doesn't, either because of some complex breaking change or because maintainers didn't have time to update?

3- When releasing patch notes/migration guides/new, updates on `nursery` crates will be included?